test/typescript_declarations/main.ts

```diff
@@ -0,0 +1,79 @@
+import type { Option$ } from "./build/dev/javascript/gleam_stdlib/gleam/option.d.mts";
+import { List, Ok, Result } from "./build/dev/javascript/typescript_declarations/gleam.mjs";
+import * as Gleam from "./build/dev/javascript/typescript_declarations/typescript_declarations.mjs"
+
+// Check add and add_alias
+const sum: number = Gleam.add_alias(Gleam.add(1, 2), 908);
```

Comment:
Use the satisfies operator please 🙏 

```ts
fn_int_int_returning_int satisfies (a: number, b:number) => number;
```

No need to call any functions, we are not testing the runtime behaviour at all.